purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -449,6 +450,58 @@ class PurchasesTest {
 
     // region purchasing
 
+    @Test
+    fun `isPersonalizedPrice value is passed through to purchaseSubscriptionOption`() {
```

Comment:
will need to duplicate each of these tests for every purchasing method for full test coverage..

Reply:
going to add the same test for all purchasing functions and probably split isPersonalizedPrice to it's own region. i think that's the only work left, so was hoping to start a review while i work on that